src/roswire/proxy/bag.py

```diff
@@ -221,6 +226,8 @@ def start(self) -> None:
                     f'__name:={self.__bag_name}']
```

Comment:
Having the option `-a` in the arguments overrides option `-e`, and that's why it's not working.

Reply:
Ah. That makes sense. Thanks for fixing this! :+1: